.atomist/editors/AddRugHandler.rug

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2016 Atomist, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@description "adds a new Rug Handler to an existing Rug Archive project"
+@tag "rug"
+@tag "atomist"
+editor AddRugHandler
+
+precondition IsRugArchive
```

Comment:
Since the handler is TypeScript, should you also add the `IsTypeScriptReady` precondition?